.NET/Microsoft.Recognizers.Text.DateTime/Utilities/TimexUtility.cs

```diff
@@ -255,6 +255,26 @@ public static TimeOfDayResolutionResult ParseTimeOfDay(string tod)
                     result.EndHour = 23;
                     result.EndMin = 59;
                     break;
+                case Constants.Breakfast:
```

Comment:
Instead of these repeated assignments, can't the cases just fall into the existing ranges?
```
case Constants.Evening:
case Constants.Dinner:
    result.Timex = Constants.Dinner;
```